libs/fairness/src/lib/Controls/PerformancePlot.tsx

```diff
@@ -216,25 +304,98 @@ export class PerformancePlot extends React.PureComponent<IPerformancePlotProps>
           )
         } as any
       ];
-      performanceChartModalHelpStrings = [
+      performanceChartCalloutHelpBarStrings = [
         localization.Fairness.Report.regressionPerformanceHowToRead
       ];
     }
 
+    const digitsOfPrecision = 1;
+
+    for (let j = 0; j < barPlotlyProps.data.length; j++) {
+      const outcomeMetricName = barPlotlyProps.data[j].name;
+      barPlotlyProps.data[j].customdata = [];
+      if (barPlotlyProps.data) {
+        for (
+          let i = 0;
+          i < this.props.dashboardContext.groupNames.length;
+          i++
+        ) {
+          const outcomeMetric = outcomeMetricName
+            ? `${outcomeMetricName}: `
+            : "";
+          const tempData = barPlotlyProps.data
+            ? barPlotlyProps.data[j]
+            : undefined;
+          const tempY = tempData && tempData.y ? tempData.y[i] : undefined;
+          const tempX = tempData && tempData.x ? tempData.x[i] : undefined;
+
+          const x =
+            tempX !== undefined && typeof tempX == "number" ? tempX : undefined;
+          const y =
+            tempY !== undefined && typeof tempY == "string"
```

Comment:
This is kind of unintuitive. What types could it have other than string? Perhaps we want to rule out such cases ahead of time.

Reply:
Datum allows for "string" | "date" | "number" | undefined